server/src/controllers/Events/resolver.ts

```diff
@@ -542,6 +542,7 @@ ${unsubscribeOptions}`,
     });
   }
 
+  @Authorized(Permission.EventEdit)
   @Mutation(() => Event)
   async updateEvent(
     @Arg('id', () => Int) id: number,
```

Comment:
```suggestion
    @Arg('eventId', () => Int) id: number,
```
In order for the authChecker to know which event this is, the argument needs to be `eventId`. The problem is that if the authChecker doesn't know this, it cannot determine if the user is allowed to access the resolver or not.